server/repositories/CaptureRepository.js

```diff
@@ -2,8 +2,8 @@ const BaseRepository = require('./BaseRepository');
 
 class CaptureRepository extends BaseRepository {
   constructor(session) {
-    super('capture_denormalized', session);
-    this._tableName = 'capture_denormalized';
+    super('reporting.capture_denormalized', session);
+    this._tableName = 'reporting.capture_denormalized';
```

Comment:
There is no need for this, since the db URL set up with the API is configured to have access to the reporting schema alone

Reply:
Ok, I added it because it wasn't working to make the requests when I ran the reporting api locally.  Is there a way we could set up the api to work locally as well?

Reply:
The easiest way would be to add DATABASE_SCHEMA=reporting as part of your local env variables.